src/java.base/share/classes/java/util/stream/Gatherer.java

```diff
@@ -314,7 +314,7 @@ static <A> BinaryOperator<A> defaultCombiner() {
      * @param <A> the type of the state of the returned finisher
```

Comment:
@AlanBateman @PaulSandoz Making the following changes following conversation on core-libs-dev, plus I noted that some of the descriptions of the Integrator type parameters were mis-targeted.